drivers/ukrtc/pl031/Config.uk

```diff
@@ -0,0 +1,5 @@
+config LIBUKRTC_PL031
+    bool "Arm PrimeCell RTC (PL031)"
+    default y if ARCH_ARM_64
```

Comment:
Being a peripheral, it is not necessary that all Arm cores have a PL031. What you can do instead is remove this line (remember in Kconfig `default n` is implied so you don't need to explicitly specify it), and in the Kconfig entries of of `KVM_VMM_QEMU` and `KVM_VMM_FIRECRACKER` add `imply LIBUKRTC_PL031`.